backend/corpora/dataset_processing/process.py

```diff
@@ -448,11 +454,19 @@ def main():
     except ProcessingCancelled:
         cancel_dataset(dataset_id)
         sys.exit(0)
-    except ProcessingFailed as ex:
-        logging.error(ex.status)
+    except ProcessingFailed:
+        logging.exception()
```

Comment:
Does this automatically capture the exception's message

Reply:
yes